### PR TITLE
  owncloud-client: 2.8.4 -> 2.9.0 

### PR DESCRIPTION
**Pull request by ebbertd**
###### Motivation for this change

Increase owncloud-client version from 2.8.4 to 2.9.0.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by ebbertd** [comment]
@qknight Could you take a look at this?

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -7885,7 +7885,7 @@ with pkgs;
 
   ovito = libsForQt5.callPackage ../applications/graphics/ovito { };
 
-  owncloud-client = libsForQt514.callPackage ../applications/networking/owncloud-client { };
+  owncloud-client = libsForQt515.callPackage ../applications/networking/owncloud-client { };
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:7888]:
```suggestion
  owncloud-client = libsForQt5.callPackage ../applications/networking/owncloud-client { };
```

Comment (ebbertd) [reply pkgs/top-level/all-packages.nix:7888]: Thank you for your suggested change. I made the change.

**Comment by ebbertd** [comment]
In the meantime a new version of the owncloud-client has become available. So I also increased the owncloud-client version and therefore then changed the PR title.

**Comment by bobby285271** [comment]
Result of `nixpkgs-review pr 135876` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>owncloud-client</li>
  </ul>
</details>

**Review by bobby285271 — APPROVED** [review]
Launches.

**Pull request merged**